framework/src/base/MooseApp.C

```diff
@@ -136,6 +137,20 @@ validParams<MooseApp>()
       false,
       "The libMesh Mesh underlying MooseMesh should always be a DistributedMesh");
 
+  params.addCommandLineParam<std::string>(
+      "split_mesh",
+      "--split-mesh [splits]",
+      "",
+      "comma-separated list of numbers of chunks to split the mesh into");
+
+  params.addCommandLineParam<std::string>("split_file",
```

Comment:
Maybe this parameter should be pluralized? `split_files`?

Reply:
Maybe, but the user only passes a single file name as the argument to that flag - so it is a bit weird no matter how you do the pluralization.

Reply:
I think I'm ok with `split_file`

Reply:
In MOOSE we call these file_bases. Perhaps that would be a better name (more in line with what users see in the `Outputs` block.

Reply:
Possibly, but There is actually a file that gets created that is named *exactly* what the user types in there.  As currently implemented, it is left to the user to type in the ".cpr" extension.  This was done to make is easy for tab completion to work if the user is trying to use an already split mesh.  I think it would be confusing to call it ``[bla]-base`` if we expect the user to type in a file extension.